Sources/TuistLoader/Models+ManifestMappers/TargetAction+ManifestMapper.swift

```diff
@@ -69,6 +69,10 @@ extension XcodeGraph.TargetScript {
             if path.pathString.contains("$") {
                 return [try generatorPaths.resolve(path: path)]
             }
+            // avoid globbing paths that point to specific files
+            if !path.pathString.contains("*") {
```

Comment:
Similarly to `if path.pathString.contains("$")` I skip globbing for paths that don't contain `*`